MANIFEST.in

```diff
@@ -0,0 +1 @@
+include nimare/resources/english_spellings.csv
```

Comment:
Can you include the full folder? I'm not sure if that should be `include nimare/resources/*` or `recursive-include nimare/resources/` or what.